src/test/kotlin/dab/poao/nav/no/ApplicationTest.kt

```diff
@@ -406,10 +404,4 @@ private fun dokarkRespons(ferdigstilt: Boolean) = """
 
 suspend fun OutgoingContent.asString() = this.toByteArray().decodeToString()
 
-private fun MockEngine.resetRequestHistory() {
-    while (requestHistory.isNotEmpty()) {
-        requestHistory.removeFirst()
```

Comment:
Denne finnes ikke hos meg, det er en List, ikke MutableList. Måtte gjøre litt endringer for at det skulle funke